CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 --------------
 
 ### Added
+- Add support for validation in field arguments [\#608 / crissi](https://github.com/rebing/graphql-laravel/pull/608)
```

Comment:
Since this code already moved, can you after rebase please make sure it's still under "Next release"? thanks!